AEPIdentity/Sources/Identity+PublicAPI.swift

```diff
@@ -143,3 +156,17 @@ import Foundation
         }
     }
 }
+
+extension CustomIdentity {
+    internal static func from(dict: [String: Any]) -> CustomIdentity? {
+        guard let type = dict[CodingKeys.type.rawValue] as? String,
+              let origin = dict[CodingKeys.origin.rawValue] as? String,
```

Comment:
I think only idType and identifier values are required, but origin is optional, so we should not fail if nil. will you double check this?

Reply:
Origin is set internally for each of the two types of CustomIdentifiers. I thinking, however, it may be better to not check any of the values, and just take them as they are given by the extension so there is only one place which validates the data.